SharkCage/CageManager/CageManager.cpp

```diff
@@ -123,13 +127,21 @@ std::wstring OnReceive(const std::wstring &message)
 	return path;
 }
 
+void StartCageDesktop(PSECURITY_DESCRIPTOR *security_descriptor)
```

Comment:
If you are derefencing the descriptor anyway a refernce would be safe: `const PSECURITY_DESCRIPTOR &security_descriptor`